Add Workers in the UI to Pages docs and bindings in Workers KV docs. (#1719)
cloudflare/cloudflare-docs: products/pages/src/content/how-to/add-custom-http-headers.md

}
```

> ## Deploying a Workers function in the UI

Reviewer: ```suggestion
## Deploying a Workers function in the dashboard
```

> The easiest way to start deploying your Workers function is typing [workers.new](https://workers.new/) in the browser. You'll sign-in to your account and automatically be directed to the Workers UI. There you can write your function or use one of the [examples](https://developers.cloudflare.com/workers/examples/) from the Workers docs. 

Reviewer: ```suggestion
The easiest way to start deploying your Workers function is by typing [workers.new](https://workers.new/) in the browser. Log into your account to be automatically directed to the Workers dashboard. From the Workers dashboard, write your function or use one of the [examples from the Workers documentation](/workers/examples/). 
```

> Click "Save and Deploy" when it's ready and set a [route](https://developers.cloudflare.com/workers/platform/routes/) in your domain's zone settings.

Reviewer: ```suggestion
Click "Save and Deploy" when your script is ready and set a [route](/workers/platform/routes/) in your domain's zone settings.
```

> For example, below is a Workers script you can copy and paste into the Workers dashboard that sets common security headers whenever a request hits your Pages URL. 

Reviewer 2: IMO link to the example page instead of dropping all the code in here. since we link to `/workers/examples` in a previous paragraph, this could be a paragraph calling out a _specific_ example ("Setting security headers")

> For example, here is a Workers script you can copy/paste into the UI that sets common security headers whenever a request hits your Pages URL. 

Reviewer: ```suggestion
For example, below is a Workers script you can copy and paste into the Workers dashboard that sets common security headers whenever a request hits your Pages URL. 
```

$ wrangler publish
``` 

After you have deployed your Worker, your desired HTTP header adjustments will take effect. While the Worker is deployed, you should continue to see the content from your Pages application as normal.

## Modify HTTP request headers in the dashboard with Transform Rules

You can set or remove HTTP request headers directly by using [Transform Rules](https://developers.cloudflare.com/rules/transform/request-header-modification/create-dashboard), removing the need to write any code with a Worker.

Go to your [account](https://dash.cloudflare.com/), select your zone, and then click the Rules tab.

   ![transform-rules-dashboard](./how-to/media/transform-rules-dashboard.png)

**Adding a custom header with a value:**

   ![transform-rules-add-custom-header](./how-to/media/transform-rules-add-custom-header.png)

**Deleting headers:**

   ![transform-rules-delete-headers](./how-to/media/transform-rules-delete-headers.png)

**Adjusting the value for an existing header:**

>    ![transform-rules-adjust-header-value](./how-to/media/transform-rules-adjust-header-value.png)

Author: I messed up image uploading...will fix in the morning.

## Modify HTTP request headers in the dashboard with Transform Rules

> You can set or remove HTTP request headers directly in the dashboard's Rules tab using [Transform Rules](https://developers.cloudflare.com/rules/transform/request-header-modification/create-dashboard). This removes the need to write any code with a Worker.

Reviewer 3: could we also add a line about where to find Transform Rules in the UI?

Go to your Cloudflare zone, click on X, etc…

Adding a custom header with a value:
![image](https://user-images.githubusercontent.com/67480168/124844782-ce87f780-df5a-11eb-8a76-f5245fbe6fe3.png)

Deleting headers:

> ![image](https://user-images.githubusercontent.com/67480168/124844616-6802d980-df5a-11eb-8eb9-e5aaa700f3ed.png)

Reviewer 3: we should host these images ourselves (i imagine this is from uploading via github editor)

You can set or remove HTTP request headers directly in the dashboard's Rules tab using [Transform Rules](https://developers.cloudflare.com/rules/transform/request-header-modification/create-dashboard). This removes the need to write any code with a Worker. 

> Here are some [examples](https://developers.cloudflare.com/rules/transform/request-header-modification/examples) of Transform Rules you can get started with.

Reviewer 3: can we actually include some screenshots of how to add the same headers that were added with the worker above?

---
$ wrangler generate projectname https://github.com/cloudflare/custom-headers-example
```

> 

Reviewer 2: make sure we put the `<Aside>` back here, it was removed at line 45
